### DISCUSSION TITLE
PIXI 7 have added "_" symbol to all constructors names.

### DISCUSSION THREAD

**Discussion by Megabyteceer** [pixijs/pixijs#9446] [General]
After switching to PIXI 7, debugger shows all PIXI objects instances typed as _Class  instead of Class

![image](https://github.com/pixijs/pixijs/assets/5096541/31738a46-226e-4e9e-9463-6acffc9dbf14)

Is it any special changes in library build process? All other libraries, usually expose constructors fine.

It is very annoying during debugging. Conditional breakpoints related to constructor name stop working. Additional Manual cleanup is required when you copy paste classes names from debugger.

All previous versions (4, 5, and 6) were fine.

Please can you try to fix constructors names at least in dev non minimized builds?

**Comment by bigtimebuddy** [comment]
Here's the shipped version of Container. 
https://cdn.jsdelivr.net/npm/@pixi/display/lib/Container.mjs

We use [esbuild](https://esbuild.github.io/) in v7 for build and it seems to be expressing this new output. I _think_ it might be related to our static properties (e.g., `defaultSortableChildren`) which were added somewhat recently.
```js
const _Container = class extends DisplayObject {
}
let Container = _Container;
export { Container };
```

This doesn't seem to affect every class. For instance [`Filter`](https://cdn.jsdelivr.net/npm/@pixi/core/lib/filters/Filter.mjs) is affected (also has static properties), but not [`Geometry`](https://cdn.jsdelivr.net/npm/@pixi/core/lib/geometry/Geometry.mjs). 

Curious! Will look into it, thanks for reporting.

**Reply by bigtimebuddy** [reply]
I see there's an issue about this here: https://github.com/evanw/esbuild/issues/1260
